src/css/_LogIn.scss

```diff
@@ -0,0 +1,45 @@
+#log-out-button {
+  background-color:$primary-color;
+  color: #38a2ea;
+  border: none;
+  display: inline;
+  float: right;
+  height: 4.6vw;
+  margin-top: 15px;
+  width: 18.6vw;
+  margin-right: 11px;
+  border-radius: 5px;
+}
+
+.login-view {
+  margin: 5% 30%;
+  background-color: $off-white-color;
+  opacity: 0.7;
+  font-size: 1.1em;
+}
+
+.login-view>h2 {
+  font-size: 1.5em;
+  color: $darkBlue-color;
+}
+
+.login-form {
+  color:$primary-color;
+  font-size: 1.1em;
+  display: flex;
+  flex-direction: column;
+  padding: 1em 4em 3em 4em;
+}
+
+.login-area{
+  color:$primary-color;
```

Comment:
Nice use of variables here for the colors, I like the name `$primary-color`, but you may want to change the names for `$darkBlue-color` or `$off-white-color` to things like `$secondary-color` or `$tertiary-color` just in case you all of a sudden decide you don't want something to be dark blue or off-white.